addons/mail/static/tests/new/chat/chat_window_tests.js

```diff
@@ -708,3 +708,41 @@ QUnit.test(
         assert.ok(isScrolledToBottom(target.querySelector(".o-mail-thread")));
     }
 );
+
+QUnit.test(
+    "chat window should remain folded when new message is received",
+    async function (assert) {
+        const pyEnv = await startServer();
+        const partnerId = pyEnv["res.partner"].create({ name: "Demo" });
+        const userId = pyEnv["res.users"].create({
+            name: "Foreigner user",
+            partner_id: partnerId,
+        });
+        pyEnv["mail.channel"].create({
+            channel_member_ids: [
+                [
+                    0,
+                    0,
+                    {
+                        fold_state: "folded",
+                        is_minimized: true,
+                        partner_id: pyEnv.currentPartnerId,
+                    },
+                ],
+                [0, 0, { partner_id: partnerId }],
+            ],
+            channel_type: "chat",
+            uuid: "channel-uuid",
+        });
+        const { env } = await start();
+        assert.hasClass(document.querySelector(".o-mail-chat-window"), "o-folded");
+
+        env.services.rpc("/mail/chat_post", {
+            context: { mockedUserId: userId },
+            message_content: "New Message",
+            uuid: "channel-uuid",
+        });
+        await nextTick();
```

Comment:
Why not use `afterNextRender`? Receiving a new message should at least update the unread counter

Reply:
Not yet re-implemented